src/AzureSqlAction.ts

```diff
@@ -127,9 +127,10 @@ export default class AzureSqlAction {
             case SqlPackageAction.Publish: 
             case SqlPackageAction.Script:
             case SqlPackageAction.DeployReport:
-                args += `/SourceFile:"${inputs.filePath}"`
+                args += `/Action:${SqlPackageAction[inputs.sqlpackageAction]} /TargetConnectionString:"${inputs.connectionConfig.ConnectionString}" /SourceFile:"${inputs.filePath}"`
+                break;
             case SqlPackageAction.DriftReport:
-                args += ` /Action:${SqlPackageAction[inputs.sqlpackageAction]} /TargetConnectionString:"${inputs.connectionConfig.ConnectionString}"`;
+                args += `/Action:${SqlPackageAction[inputs.sqlpackageAction]} /TargetConnectionString:"${inputs.connectionConfig.ConnectionString}"`;
```

Comment:
Add a space here too. Thank you!

Reply:
I have a commit up adding that space. My only concern is that before I put up this PR there was no space at the beginning. Because we declare `args` inside of that method and first assign to it inside of that case statement, it makes sense to me why we do not need that space. Do you have any concern with the new leading whitespace in that context?

Thanks!

Reply:
Oh sorry I was looking at the wrong code, yeah it makes sense to have no space then. This is the argument passed to sqlpackage and `/Action` is the start of it. You can remove the space. Sorry again for the confusion.

Reply:
No worries at all, thanks for the follow up and assistance!

Reply:
Ok, that change is up. Thanks again.